README.md

```diff
@@ -105,6 +105,69 @@ The navigation system comprised SLAM provided by the Cartographer online node an
 The global plan was generated by the default global planner known simply as global_planner.
 </p>
 
+### Control
+
+The robot needs to be able to navigate unknown environments, a robust and reliable control system is needed to achieve this.
+
+![](https://github.com/D7039E-E7032E/Project-Report/blob/Controll-delen-till-rapport/images/Control_Str.svg)
+
+In the illustration above the structure of the control system an its supporting components. 
+
+#### Global Navigator
+
+The purpose of the global navigator is to find a path the robot should follow to reach its goal. The the goal points is set manually and when the robot reaches it, a new one will be chosen by the operator. It will use information from the global map, a map of the combined scans from all robots. This is implemented using the already existing ROS package move_base that calculates a global path for the robot using a form of A-star algorithm. 
+
+The Global Navigator uses maps generated from cartographer, which is a slow process, combined with A-star not being a particular fast algorithm. This makes the Global Navigator a rather slow. This is solved with a Local Navigator.
+
+#### Local Navigator
+
+The Local Navigator's purpose is to avoid obstacles and keeping an optimal distance to walls, this to give optimal scanning quality. It will follow the direction of the path given by the layout navigator.
+
+![Force](https://github.com/D7039E-E7032E/Project-Report/blob/Controll-delen-till-rapport/images/Force.svg)
+
+A custom ROS node was implemented to solve this. The node operates by giving all point seen by the lidar a virtual force, see equation above, which when combined creates an heat map. The node then solves the gradient of the heat map in the position of the robot, as in equation below. The value of *lp* is the distance to the point, and the values *c* and *n* is tuning-parameters, which was tuned manually. The resulting vector will then point in a direction away from all obstacles.
```

Comment:
use *l<sub>p</sub>* sub-tags for describing index *p* of *l*. sup-tags gives superscript output.

Reply:
vet inte hur man gör detta, du får göra det.

Reply:
man gör som i html, l<sub>p</sub> = `l<sub>p</sub>`